RELEASE_NOTES.md

```diff
@@ -2,6 +2,17 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+# 0.16.1 (October 5, 2023)
+
+- 🍎 iOS crash fix
```

Comment:
This isn't actually specific to iOS I don't think?

"Fix for a rare crasher seen by large apps."